doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -83,6 +83,9 @@ Bluetooth mesh
 * Updated:
 
   * :ref:`bt_mesh_dm_srv_readme` and :ref:`bt_mesh_dm_cli_readme` model IDs and opcodes have been updated to avoid conflict with Simple OnOff Server and Client models.
+  * :ref:`bt_mesh_sensors_readme` now use an updated API using :c:struct:`bt_mesh_sensor_value` instead of :c:struct:`sensor_value` to represent sensor values.
+    This makes it possible to accurately represent all encodable sensor values.
+    The old APIs based on :c:struct:`sensor_value` are deprecated, but are still available for backward compatibility, and can be enabled by using :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`.
```

Comment:
Why "The old APIs based on <ins>the</ins> `sensor_value` are deprecated"? It reads a bit weird to me since `sensor_value` is a type. As weird as e.g. "The old APIs based on <ins>the</ins> `float` are deprecated".

Reply:
Sorry, I missed adding type in the end.
So, either based on :c:struct:... or based on the :c:struct:... type, your choice. You can keep it as it was with no article in the front, it's ok for me.